### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by rymaju**
- Small text update in Jumpstart page
- Added Youtube link to footer
- Fixed various ESLint issues

### CODE REVIEW

**Comment by rymaju** [comment]
Also relinted, not sure why but it affected many files.

File: src/pages/projects/ProjectCard.tsx [hunk 1]
```diff
@@ -1,75 +1,87 @@
...
+  paragraph,
+  appDesc,
+  path,
+  abbr,
+  learnMore,
+}) => {
+  const classes = useStyles();
+  return (
+    <Grid item md={6}>
+      <Card className={classes['project-card']}>
```
Comment (jackblanc) [inline src/pages/projects/ProjectCard.tsx]: 💭 can we not access this with `.project-card`?

File: src/pages/projects/ProjectJSON.tsx [hunk 2]
```diff
@@ -1,30 +1,32 @@
 export default {
```
Comment (jackblanc) [inline src/pages/projects/ProjectJSON.tsx]: 🌱 this should be type-checked

**Review by jackblanc — APPROVED** [review]

File: src/pages/projects/ProjectCard.tsx (hunk 1 above)
Comment (rymaju) [reply src/pages/projects/ProjectCard.tsx]: Oh my, this is very not correct, lets fix that first

File: src/pages/projects/ProjectJSON.tsx (hunk 2 above)
Comment (rymaju) [reply src/pages/projects/ProjectJSON.tsx]: Yep @54skyxenon do you have time to fix these?

Comment (54skyxenon) [reply src/pages/projects/ProjectJSON.tsx]: Yeah sure thing, just fixed @rymaju

File: src/pages/projects/ProjectJSON.tsx [hunk 3]
```diff
@@ -1,30 +1,39 @@
...
+export interface Project {
+  readonly title: string;
+  readonly paragraph: string;
+  readonly appDesc: string;
+  readonly path: string;
+  readonly abbr: string;
+  readonly learnMore: string;
+}
+
+export const projects : Project[] = [
```
Comment (jackblanc) [inline src/pages/projects/ProjectJSON.tsx:10]: ⛏️ why is there a space after projects? Also on line 12 there are double quotes.

**Review by jackblanc — COMMENTED** [review]
It looks like lint didn't run on the projects data file? Not sure why 👀

**Pull request merged**